src/Admin/Views/Shared/_Layout.cshtml

```diff
@@ -61,9 +61,12 @@
                         </li>
                         @if (!GlobalSettings.SelfHosted)
                         {
-                            <li class="nav-item" active-controller="Providers">
-                                <a class="nav-link" asp-controller="Providers" asp-action="Index">Providers</a>
-                            </li>
+                            @if (AccessControlService.UserHasPermission(Permission.Provider_List_View))
```

Comment:
suggestion(non-blocking): same question about consistency and using a variable instead.